math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -3242,6 +3242,274 @@ RVec<typename RVec<T>::size_type> Enumerate(const RVec<T> &v)
    return ret;
 }
 
+/**
+ * \brief Produce RVec with N evenly-spaced entries from start to end.
+ *
+ * This function generates a vector of evenly spaced values, starting at \p start and (depending on the
+ * \p endpoint parameter) either including or excluding \p end. If \p endpoint is true (default),
+ * the vector contains \p n values with \p end as the final element, and the spacing is computed as
+ * \f$\text{step} = \frac{\text{end} - \text{start}}{n-1}\f$. If \p endpoint is false,
+ * the sequence consists of n values computed as if there were n+1 evenly spaced samples, with the final
+ * value (\p end) omitted; in this case, \f$\text{step} = \frac{\text{end} - \text{start}}{n}\f$.
+ *
+ * The function is templated to allow for different arithmetic types. The return type \c Ret_t, if
+ * not explicitly specified, is determined as follows: if \p T is a floating point type, that type is used;
+ * otherwise, the arithmetic is performed using \c double.
+ *
+ * \tparam T Type of the start and end value. Default is double.
+ * \tparam Ret_t Return type used for arithmetic, which, if not explicitly specified
+ * in the template, is \p T if that is a floating point type, or double otherwise.
```

Comment:
```suggestion
 * The function is templated to allow for different return types. The return type \c Ret_t, if
 * not explicitly specified, is determined as follows: if \p T is a floating point type, that type is used;
 * otherwise, the return type is \c double.
 *
 * \tparam T Type of the start and end value. Default is double.
 * \tparam Ret_t Return type used, which, if not explicitly specified
 * in the template, is \p T if that is a floating point type, or double otherwise.
```